src/main/java/frc/robot/commands/ToggleJoint.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Intake.PivotState;
+
+public class ToggleJoint extends CommandBase {
+
+  private Intake intake;
+  private double motorPower;
+
+  public ToggleJoint(double power) {
+    motorPower = power;
+
+    intake = Intake.getInstance();
+
+    addRequirements(intake);
+  }
+
+  /**
+   * If PivotState is down, send negative power. Otherwise, send positive power.
+   */
+  @Override
+  public void execute() {
+    if (intake.getPivotState() == PivotState.down) {
+      intake.runJoint(-motorPower);
+    } else {
+      intake.runJoint(motorPower);
+    }
+  }
+
+  /**
+   * If PivotState is down, return state of limit switch, which should be false.
```

Comment:
Bad comment.  You are describing the code in words, describe why it happens not how it happens